src/Mages.Core.Tests/IntellisenseTests.cs

```diff
@@ -12,7 +12,7 @@ public void EmptyScopeAndSourceYieldsKeywords()
         {
             var source = "";
             var engine = new Engine();
-            engine.Globals.Clear();
+            engine.Scope.Clear();
```

Comment:
We want to clear the globals (API layer) - not the scope. The scope (user layer) is initially empty!